.taskcluster.yml

```diff
@@ -107,7 +107,7 @@ tasks:
                       level:
                           $if: 'tasks_for in ["github-push", "github-release", "action", "cron"] && repoUrl == "https://github.com/mozilla-mobile/mozilla-vpn-client"'
                           then: '3'
-                          else: '1'
+                          else: '3'
```

Comment:
We currently do not have windows worker that pick this up for level-1 tasks. 
https://firefox-ci-tc.services.mozilla.com/provisioners/mozillavpn-1/worker-types/b-win2012